packages/cfd/src/Components/__tests__/cfd-mt5-demo-account-display.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CFDMT5DemoAccountDisplay from '../cfd-mt5-demo-account-display';
+import { CFD_PLATFORMS } from '@deriv/shared';
+
+//mock cfd-account-card
+jest.mock('../cfd-account-card', () => ({
+    CFDAccountCard: () => <div>CFDAccountCard</div>,
+}));
+
+describe('CFDMT5DemoAccountDisplay', () => {
+    const mock_standpoint = {
+        financial_company: 'financial_company',
+        gaming_company: 'gaming_company',
+        iom: false,
+        malta: false,
+        maltainvest: false,
+        svg: true,
+    };
+
+    const mock_props = {
```

Comment:
```suggestion
    const mock_props: React.ComponentProps<typeof CFDMT5DemoAccountDisplay> = {
```
@hamza-deriv Just for better type safety and auto-completion 🙇🏻